### DISCUSSION TITLE
Shortcuts in chrome settings

### DISCUSSION THREAD

**Discussion by ghost** [sienori/simple-translate#272] [Q&A] [answered]
Hi, 

in Firefox there are keyboard shortcuts to translate the page, for opening the popup ecc.

But in chrome why aren't there?

Thank you for your fantastic work

**Comment by sienori** [comment, marked as answer]
In Chrome, you can set shortcuts in `chrome://extensions/shortcuts`. The reason why you cannot set shortcuts on the extension settings page like firefox is because Chrome does not support the API.
![image](https://user-images.githubusercontent.com/29514424/110617797-6fe38480-81d9-11eb-8596-bafd63416209.png)
